lib/web_ui/lib/src/engine/canvaskit/native_memory.dart

```diff
@@ -130,9 +129,11 @@ class CountedRef<R extends StackTraceDebugger, T extends Object> {
   /// Creates a counted reference.
   CountedRef(T nativeObject, R debugReferrer, String debugLabel) {
     _ref = UniqueRef<T>(this, nativeObject, debugLabel);
-    if (assertionsEnabled) {
+    assert(() {
       debugReferrers.add(debugReferrer);
-    }
+      return true;
+    }());
+    // TODO: @yjbanov what's the purpose of this? It's always true.
     assert(refCount == debugReferrers.length);
```

Comment:
@yjbanov, a question for you :)

During construction, `refCount` is initialized to `1` and `debugReferrers` is initialized to an empty set then one item is added.